src/main/java/com/example/module2/Address.java

```diff
@@ -0,0 +1,81 @@
+package com.example.module2;
+
+public class Address {
+    private String country;
+    private String city;
+    private int postcode;
```

Comment:
Please change type to `String`. Postal codes could be like `T0M 0A0` : )
And rename to `postCode` or `postalCode`.